odl/tomo/backends/astra_cuda.py

```diff
@@ -100,7 +100,8 @@ def call_forward(self, vol_data, out=None):
         if self.geometry.ndim == 2:
             out[:] = self.out_array
         elif self.geometry.ndim == 3:
-            out[:] = np.rollaxis(self.out_array, 0, 3)
+            out[:] = np.rollaxis(self.out_array, 0, 3).reshape(
```

Comment:
I think `reshape` just reshuffles strides so it should be for free large, but if there's overhead I'll make this step only when necessary.